### DISCUSSION TITLE
#17. VATZ biweekly meeting at 2022-09-14 08:00:00 -06:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#239] [General]
### 1. Overall
<img width="1211" alt="image" src="https://user-images.githubusercontent.com/6308023/187747225-7a21c816-1ed9-4a81-9c6a-ab14cbb2aaa7.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
15|  4/16 | 20
16 | 2/18 | 11

---

- Finished
  - #120
  - #229 
    
- Move to Sprint 17
   - #222 
   - #85 
   - #87 
   - #119  
   - #121 
   - #122 
   - #146 
   - #174 
   - #175 
   - #187 
   - #197 
   - #202 
   - #225 
   - #226 
   - #228
   - #238
     
---

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/248
   - https://github.com/dsrvlabs/vatz/issues/87 
- https://github.com/dsrvlabs/vatz/issues/238
- https://github.com/dsrvlabs/vatz/issues/240
   - https://github.com/dsrvlabs/vatz/issues/174

**Comment by rootwarp** [comment]
# 준교
- https://github.com/dsrvlabs/vatz-proto/issues/9
- https://github.com/dsrvlabs/vatz/issues/222
- https://github.com/dsrvlabs/vatz/issues/220

**Comment by kim201212** [comment]
# 형기 

- No issue.

**Comment by gnongs** [comment]
## 진홍
- https://github.com/dsrvlabs/vatz/issues/226

**Comment by heejin-github** [comment]
## 희진
- #242 
  - #249 
- #236 
  - #245

**Comment by skonhwang** [comment]
# 승곤
- https://github.com/dsrvlabs/vatz/issues/244
- https://github.com/dsrvlabs/vatz-plugin-mevboost
- https://github.com/dsrvlabs/schellingpoint-project/issues/20#issuecomment-1246490311

**Comment by meetrick** [comment]
# 황제
- No update
